code/datums/trading/trade.dm

```diff
@@ -122,7 +122,7 @@
 	if(!trading_items[trading_items[trading_num]])
 		var/type = trading_items[trading_num]
 		var/value = get_value(type)
-		value = round(rand(90,110)/100 * value) //For some reason rand doesn't like decimals.
```

Comment:
Way too low. Keep it between 100 and 80

Reply:
Code amended as per suggestion, resolving.